pkg/apis/config/defaults_test.go

```diff
@@ -35,21 +35,22 @@ func TestDefaultsConfigurationFromFile(t *testing.T) {
 	cm, example := ConfigMapsFromTestFile(t, DefaultsConfigName)
 
 	if _, err := NewDefaultsConfigFromConfigMap(cm); err != nil {
-		t.Errorf("NewDefaultsConfigFromConfigMap(actual) = %v", err)
+		t.Error("NewDefaultsConfigFromConfigMap(actual) =", err)
```

Comment:
(nit) Missing trailing space before the error.
Also the name of the test will be in the report, so `t.Error("actual: ", err)` should be sufficient.
```suggestion
		t.Error("actual: ", err)
```

Reply:
t.Error does add a space. I know it is not consistent with the other methods, see: https://play.golang.org/p/bty6llJYM-5

Reply:
Right, my bad. The rest should remain valid though.